File: SWEA/swea_9997_minimulism_clock.java
import java.util.Scanner;
import java.io.FileInputStream;


class swea_9997_minimulism_clock {
	public static void main(String args[]) throws Exception {
        // Scanner sc = new Scanner(System.in);
        Scanner sc = new Scanner(new FileInputStream("../input.txt"));

		int T;
		T=sc.nextInt();
		for(int test_case = 1; test_case <= T; test_case++) {
            int degree = sc.nextInt();

            int hour = degree / 30;
            int minute = (int)((degree - (hour * 30)) / 0.5);

            System.out.printf("#%d %d %d", test_case, hour, minute);
            System.out.println();
		}
	}
}

File: SWEA/swea_9997_minimulism_clock.py
# swea 9997 미니멀리즘 시계

import sys
sys.stdin = open('../input.txt', 'r')

T = int(input())
for test_case in range(T):
    degree = int(input())

    hour = degree // 30
    minute = int((degree - (30 * hour)) / 0.5)

    print(hour, minute)